charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -1571,23 +1571,26 @@ spec:
                                 enabled:
                                   description: Whether or not to enable the Presto-Prometheus connector.
                                   type: boolean
-                                uri:
-                                  description: URI for Prometheus.
-                                  type: string
-                                  format: uri
-                                chunkSizeDuration:
-                                  description: Default size of each Prometheus query chunk.
-                                  type: string
-                                maxQueryRangeDuration:
-                                  description: Default range for Prometheus query, divided into chunkSizeDuration chunks.
-                                  type: string
-                                cacheDuration:
-                                  description: Time to live for Presto checking connector properties values.
-                                  type: string
-                                bearer:
+                                config:
+                                  type: object
+                                  properties:
+                                    uri:
+                                      description: URI for Prometheus.
+                                      type: string
+                                      format: uri
+                                    chunkSizeDuration:
+                                      description: Default size of each Prometheus query chunk.
+                                      type: string
+                                    maxQueryRangeDuration:
+                                      description: Default range for Prometheus query, divided into chunkSizeDuration chunks.
+                                      type: string
+                                    cacheDuration:
+                                      description: Time to live for Presto checking connector properties values.
+                                      type: string
+                                auth:
                                   type: object
                                   properties:
-                                    tokenFile:
+                                    bearerTokenFile:
                                       description: Path to token file for bearer authentication to Prometheus.
                                       type: string
                                 useServiceAccountToken:
```

Comment:
This needs to be under `prometheus.auth`.

Reply:
Fixed in: 438ae4ea